Automattic/wp-calypso
# Jetpack Cloud: Add credentials warnings

#### Changes proposed in this Pull Request

* Removes the `MissingCredentialsWarning` component and replaces with logic directly in `DailyBackupStatus`.
* Adds credentials warnings to DailyBackupStatus
* Adds credentials warnings to ActivityCard when credentials are not present
* Fixes `getDoesRewindNeedCredentials()` selector to actually return a valid status

#### Testing instructions

* Remove credentials from a working test site. Check DailyBackupStatus as well as ActivityCards to ensure that the warnings show as appropriate, lead to the proper settings page, do not cause any console errors, and look good per the designs.
